cashaddr.md

```diff
@@ -0,0 +1,117 @@
+# Address format for Bitcoin Cash
+
+Version 1.0, 2017-10-13
+
+## Abstract
+
+This document describes proposed address format to be used on Bitcoin Cash. It is a base32 encoded format using BCH codes as checksum and that can be used directly in links or QR codes.
```

Comment:
would be good to give a reference to type out the acronym for BCH and give a reference link when the acronym is first introduced in this spec. It's easily confused with current ticker used widely for Bitcoin Cash.

Bose–Chaudhuri–Hocquenghem (BCH)

ref. https://en.wikipedia.org/wiki/BCH_code